### PR TITLE
feature(unlock-app): Checkout create account

### PR DESCRIPTION
**Pull request by julien51**
# Description

Adding more hints that users should _not_ use unlock accounts if they have a walle!

# Checklist:

- [ ] 1 PR, 1 purpose: my Pull Request applies to a single purpose
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have updated the docs to reflect my changes if applicable
- [ ] I have added tests (and stories for frontend components) that prove my fix is effective or that my feature works
- [ ] I have performed a self-review of my own code
- [ ] If my code involves visual changes, I am adding applicable screenshots to this thread

### CODE REVIEW

File: unlock-app/src/components/interface/checkout/Connected.tsx [hunk 1]
```diff
@@ -217,6 +219,7 @@ export function Connected({
           send('UNLOCK_ACCOUNT')
         }}
         authenticateWithProvider={authenticateWithProvider}
+        title="Have a crypto wallet?"
```
Comment (kalidiagne) [inline unlock-app/src/components/interface/checkout/Connected.tsx:222]:
```suggestion
        title={title}
```
we can use title also here?

**Comment by kalidiagne** [comment]
LG! please check the comment

File: unlock-app/src/components/interface/checkout/UnlockAccount/EnterEmail.tsx [hunk 2]
```diff
@@ -60,18 +64,26 @@ export function EnterEmail({ unlockAccountService }: Props) {
...
+            className="underline text-ui-main-500"
           >
-            <span>Learn more</span>
-            <ExternalLinkIcon />
-          </a>
+            Unlock account
+          </Link>
+          . If you already have a crypto wallet, please connect it.
         </p>
+        <section className="ml-0.5 text-sm mt-auto	">
```
Comment (searchableguy) [inline unlock-app/src/components/interface/checkout/UnlockAccount/EnterEmail.tsx:79]: What do you think about de-emphaizing this using gray text? 🤔

**Review by searchableguy — APPROVED** [review]

File: unlock-app/src/components/interface/checkout/UnlockAccount/EnterEmail.tsx (hunk 2 above)
Comment (searchableguy) [reply unlock-app/src/components/interface/checkout/UnlockAccount/EnterEmail.tsx:79]: Also having a hierarchy here. Did you know could be bolder even if same size?

File: unlock-app/src/components/interface/checkout/Connected.tsx (hunk 1 above)
Comment (julien51) [reply unlock-app/src/components/interface/checkout/Connected.tsx:222]: No there is no `title` in this component.

File: unlock-app/src/components/interface/checkout/UnlockAccount/EnterEmail.tsx (hunk 2 above)
Comment (julien51) [reply unlock-app/src/components/interface/checkout/UnlockAccount/EnterEmail.tsx:79]: Great ideas. Let me tweak that.

**Comment by julien51** [comment]
Screenshot 2023-02-20 at 5 34 02 PM

**Pull request merged**